src/main/scala/lenthall/config/ScalaConfig.scala

```diff
@@ -8,6 +10,7 @@ object ScalaConfig {
   implicit class EnhancedScalaConfig(val config: Config) extends AnyVal {
     def getConfigOption(key: String): Option[Config] = getOption(key, config.getConfig)
     def getStringOption(key: String): Option[String] = getOption(key, config.getString)
+    def getPathOption(key: String): Option[Path] = getStringOption(key) map { Paths.get(_) }
```

Comment:
I actually, for less than a second, thought a similar thing as @cjllanwarne. While we name our params `key` in each of these implicits methods, typesafe config actually uses [`path`](https://github.com/typesafehub/config/blob/v1.2.1/config/src/main/java/com/typesafe/config/Config.java#L12-L13) as all the parameter names.